clouddriver-google/src/main/groovy/com/netflix/spinnaker/clouddriver/google/provider/agent/GoogleRegionalServerGroupCachingAgent.groovy

```diff
@@ -281,10 +283,11 @@ class GoogleRegionalServerGroupCachingAgent extends AbstractGoogleCachingAgent i
       }
       serverGroup.instances.clear()
 
-      serverGroup.asg.loadBalancerNames.each { String loadBalancerName ->
-        loadBalancerKeys << Keys.getLoadBalancerKey(region,
-                                                    accountName,
-                                                    loadBalancerName)
+      serverGroup.asg.get(REGIONAL_LB_NAMES).each { String loadBalancerName ->
```

Comment:
Since this logic looks the same is `GoogleZonalServerGroupCachingAgent` and `GoogleRegionalServerGroupCachingAgent`, you can just define some helper methods in `GoogleZonalServerGroupCachingAgent` and call them from `GoogleRegionalServerGroupCachingAgent`. We are already doing this in several spots.